.github/workflows/helm-chart.yml

```
@@ -0,0 +1,107 @@
+        run: |
+          if helm template phase-server deploy/helm/phase-server \
+               --namespace phase -f /tmp/scaleout.yaml > /tmp/no-crd.yaml 2>/tmp/no-crd.err; then
+            echo "expected the render to fail without monitoring.coreos.com/v1"; exit 1
+          fi
+          grep -q 'prometheus-operator CRDs' /tmp/no-crd.err
+
+      # The escape hatch stays open: with prometheusRule.enabled=false the operator
+      # is not required at all, because the recording rule is supplied externally
+      # (prometheus-adapter externalRules). That must still render on a bare cluster.
+      - name: helm template (scale-out, external recording rule, no CRDs)
+        run: |
```

actionlint (1.7.12)

[error] shellcheck reported issue in this script: SC2251:info:7:1: This ! is not on a condition and skips errexit. Use `&& exit 1` instead, or make sure $? is checked

(shellcheck)